### ISSUE TITLE
Exit Code 1 

### ISSUE THREAD

**Issue by FafoPz** [AllTheMods/ATM-8#490]
[error minecraft all.txt](https://github.com/AllTheMods/ATM-8/files/10280244/error.minecraft.all.txt)

I have no idea whats happening, i am using Java Development kit 19.0.1 (64-bit)

**Comment by P0kys** [comment]
How much RAM do you have allocated? It seems like you need more

**Comment by FafoPz** [comment]
8 Gb, it creashes when entering world

**Comment by P0kys** [comment]
Try allocating maybe 12 (or as much as you can) and see whether that helps

**Comment by FafoPz** [comment]
it continues to crash

**Comment by P0kys** [comment]
You say it's crashing when loading your world? What about when you create a new world? Does it still crash?
Also, your log from the beginning says that you have 10Gb allocated (10016m to be exact). How much RAM does your PC have overall?

**Comment by Sw3kt** [comment]
Hello, I am also having the same issue. My system has 16gb in total and allocating 10gb or 12gb just doesn't seem to fix the constant crashing. Trying to run the game lower than 10gb won't even let me load into the world. The error keeps saying that "There is insufficient memory for the Java Runtime Environment to continue". Even when the world is loaded, I can't seem to play for an hour before the crash happens. All of my drivers are up to date. I tried older versions of the modpack and I still get the same crashes.

**Comment by Sw3kt** [comment]
I might have a solution for this crashing. Someone told me to remove Spark but I was still getting crashes when loading into new chunks. After looking further, a redditor suggested to remove Rubidium and Oculus and so I did. I haven't had as many crashes as I did. I tested on modpack version 1.0.5 without Spark, Oculus, and Rubidium and it is running a lot better for me. I don't play on a server so I don't know how this will affect it. You may notice some stuttering here and there after removing these mods. I have a decent rig so my fps didn't dip below 100 frames. I still need to try it on 1.0.7 to see if these three mods are the issue. I installed Canary mod and Starlight which improved performance to some degree.

*The only downside to this solution is that you won't be able to play with shaders!*

TLDR; Remove Spark, Oculus, and Rubidium
